src/decorator/string/IsPhoneNumber.ts

```diff
@@ -4,18 +4,30 @@ import { parsePhoneNumberFromString, CountryCode } from 'libphonenumber-js';
 
 export const IS_PHONE_NUMBER = 'isPhoneNumber';
 
+type Options = { region?: CountryCode; strategy?: 'valid' | 'possible' };
+
+type RegionOrOptions = CountryCode | Options;
+
+function normalizeOptions(regionOrOptions?: RegionOrOptions): Options | undefined {
+  return 'string' === typeof regionOrOptions ? { region: regionOrOptions } : regionOrOptions;
```

Comment:
```suggestion
  return typeof regionOrOptions === 'string' ? { region: regionOrOptions } : regionOrOptions;
```
I would say this is the preferred syntax for this condition.

Reply:
Sure. I tend to use [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) but I understand these may not be to everybody's taste.